setup.cfg

```diff
@@ -70,3 +71,10 @@ exclude_lines =
   pragma: py{ignore_python_version}
   # Don't complain about IPython completion helper
   def _ipython_key_completions_
+
+[default_paths]
+# Set these paths to the location where you want this machine to store the
+# default data and reference libraries, or leave blank if you'd like to use the
+# default (astropy cache) machinery to handle this for you:
+data_root =
+ref_lib_data =
```

Comment:
These two variables can be set with the path to the directories that you want to use, and when you `python setup.py install` the package, it will use those as the default lookup locations.

Reply:
Good, but now the tests fail... maybe they don't like blank assignments in the setup.cfg?